eZ/Bundle/EzPublishIOBundle/BinaryStreamResponse.php

```diff
@@ -133,6 +137,11 @@ public function setContentDisposition( $disposition, $filename = '', $filenameFa
      */
     public function prepare( Request $request )
     {
+        if ( !$this->file instanceof BinaryFile )
```

Comment:
> prepare method use directly $this->file
Not really. `prepare()` doesn't accept any argument, meaning that you need to go through either setFile or the constructor. There is no way that I can see that this class can be instanciated without a BinaryFile object.

> imagine deleted file for any reason
Now that is a valid point. But checking for the type won't help on this, we'd have to do something else.